### PR TITLE
Add nasa9084 to sig-docs-l10n-admins

### LINKED ISSUE #996: Request admin permission for docs-l10n
### Organization or repository
kubernetes/website, docs-l10n

### Users affected
@nasa9084

### Describe the issue
I'm an owner of docs-ja project, to manage the project (rebase & force-push before PR to master), I need permission to the repository (I don't know why only I do not have the permission but other approvers for this project have...)

### PR DESCRIPTION
**Pull request by nasa9084**
fixes #996

### CODE REVIEW

File: config/kubernetes/sig-docs/teams.yaml [hunk 1]
```diff
@@ -187,6 +187,7 @@ teams:
     - tnir # Japanese
     - zacharysarah # English
     - zhangxiaoyu-zidif # Chinese
+    - nasa9084 # Japanese
```
Comment (nikhita) [inline config/kubernetes/sig-docs/teams.yaml]: The list of members needs to be alphabetically sorted :)

**Comment by nikhita** [comment]
@nasa9084 can you also squash the commits into a single commit? thanks!

**Comment by nasa9084** [comment]
I see.

**Comment by nasa9084** [comment]
squashed!

**Comment by zacharysarah** [comment]
/lgtm
/approve

@jimangel 👋 This is how we manage branch permissions for localizations in k/website.

**Pull request merged**